doc/release-notes/upgrading.en.rst

```diff
@@ -85,3 +85,8 @@ Metrics
 ------------------
 
 The HTTP connection metric proxy.process.http.dead_server.no_requests has been renamed to proxy.process.http.down_server.no_requests.
+
+Logging
+------------------
+
+The ``cqtx`` log field has been deprecated.
```

Comment:
"deprecated but can still be replaced by ``cqhm pqu cqpv``"

Reply:
Yeah it's good to mention the replacement options. Added that.